src/Persistence.Tests/Yaml/RoundTripTests.cs

```diff
@@ -92,6 +92,7 @@ public void Screen_With_Component_Instance(string path, bool isControlIdentifier
         customControl.Should().NotBeNull();
         customControl.Name.Should().Be("This is custom component");
         customControl.ComponentName.Should().Be("ComponentDefinition_1");
+        customControl.ComponentLibraryUniqueName.Should().Be("MyComponentLibrary");
```

Comment:
Can we add test when this field is optional?

Reply:
good idea